mautic/mautic
# update french regions (departements)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 🟥 
| Deprecations?                          | 🟥 
| BC breaks? (use the c.x branch)        | ✅ 
| Automated tests included?              | Not relevant
| Related user documentation PR URL      | Not relevant
| Related developer documentation PR URL | Not relevant
| Issue(s) addressed                     | 



#### Description:
This PR updates the list of French region relying on ISO_3166-2. This fixes 3 current issues:
1. Add missing regions
2. Update oversea french regions (add and rename one properly)
3. Reorder them in ISO order (region number instead of alphabetical)

#### Steps to test this PR:
* Check list if same as ISO_3166-2
* Check dropdown list of region is still working (in contact creation page for example)

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10802"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
